exercises/practice/twelve-days/twelve_days_test.cpp

```diff
@@ -0,0 +1,116 @@
+#include "twelve_days.h"
+#ifdef EXERCISM_TEST_SUITE
+#include <catch2/catch.hpp>
+#else
+#include "test/catch.hpp"
+#endif
+
+TEST_CASE("first day a partridge in a pear tree", "[c0b5a5e6-c89d-49b1-a6b2-9f523bff33f7]") {
+    std::string_view const expected = "On the first day of Christmas my true love gave to me: a Partridge in a Pear Tree.";
```

Comment:
My bad for suggesting this in the previous iteration. A string_view of string literal should be constexpr:
```suggestion
    constexpr std::string_view expected = "On the first day of Christmas my true love gave to me: a Partridge in a Pear Tree.";
```